packages/fx-core/resource/package.nls.json

```diff
@@ -533,6 +533,7 @@
   "core.common.CancelledMessage": "Operation is canceled.",
   "core.common.SwaggerNotSupported": "Swagger 2.0 is not supported. Convert it to OpenAPI 3.0 first.",
   "core.common.SpecVersionNotSupported": "OpenAPI version %s is not supported. Use version 3.0.x.",
+  "core.common.AddedAPINotInOriginalSpec": "APIs added to the project should originate from the original OpenAPI description document.",
```

Comment:
Revised: 
APIs added to the project need to originate from the original OpenAPI description document.